lib/conjur/cli.rb

```diff
@@ -91,6 +94,26 @@ def init!
         load_plugins
         commands_from 'conjur/command'
       end
+
+      def command_version_compatible? command
+        version_compatible = false
+        if command.instance_variable_defined?(:@conjur_min_version)
+          begin
+            appliance_version = Conjur::API.appliance_info["services"]["appliance"]["version"]
```

Comment:
You can get this with `Conjur::API.service_version 'appliance'` and it'll already be a `Semantic::Version`.